### ISSUE TITLE
Data scraping function not working

### ISSUE THREAD

**Issue by omkarxx** [JoshWeiner/NBA_Game_Prediction#3]
The functions defined to scrape data using the URLs saved in url_data json file are not working.
The functions just save the empty csv file.
please look into the functions asap.

![image](https://user-images.githubusercontent.com/57015056/145769928-b6a317e0-f30b-4383-b40d-2e1f4e20cf7b.png)

**Comment by barbaraon** [comment]
Could  you solve it ? @omkarxx

**Comment by JoshWeiner** [comment]
Hi all, sorry to hear about this problem you're running into.

We accessed data from Synergy Sports with funding from the University of Pennsylvania. They, through their Division 1 athletic programs had signed a contract with Synergy Sports through which we also received access.
As an unfortunate consequence of this, we are not at liberty to provide public access to our dataset- nor is it straightforward to create an account with Synergy Sports as an individual. This is likely the wall you all were running into.

However, we have compiled a list of alternative datasets through which one can replicate and improve upon our data. More work may be required to scrape certain sites than can be achieved by replacing the url in our notebook. Hope these help:
https://www.basketball-reference.com/leagues/
https://www.kaggle.com/datasets/wyattowalsh/basketball
https://www.kaggle.com/datasets/nathanlauga/nba-games
https://www.kaggle.com/datasets/drgilermo/nba-players-stats
